edxadapt/edxadapt.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import pkg_resources
+
+from django.template import Template, Context
+from HTMLParser import HTMLParser
+from xblock.core import XBlock
+from xblock.fields import Scope, Boolean, Dict, List, String
+from xblock.fragment import Fragment
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+from student.models import anonymous_id_for_user, User
+
+
+_ = lambda text: text
+log = logging.getLogger(__name__)
+html_parser = HTMLParser()
+
+
+class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
+    """
+    This XBlock automatically registers students in the EdxAdapt instance
+    using parameters configured by the teaching staff: params, skills, edx_adapt_api_url
+    """
+
+    has_author_view = True
+
+    display_name = String(
+        default='Adaptive Learning',
```

Comment:
This field actually is not being displayed to learners. The only thing they see in this xblock is a green or red status mark.

It does make sense to change xblock title to something more descriptive though.
Like `"Adaptive Learning Enrollment"` since this is what the xblock actually does.